docs/api/app.md

```diff
@@ -22,13 +38,13 @@ app(props)
 
 There are only a handful of props you can use to configure your app.
 
-| Prop                            | Type                                                                      | Required?
-| ------------------------------- | ------------------------------------------------------------------------- | ---------
-| [init](#init)                   | [State](../architecture/state.md) or [Action](../architecture/actions.md) | :100:
-| [view](#view)                   | [View](../architecture/views.md)                                          | :100:
-| [node](#node)                   | DOM element                                                               | :100:
-| [subscriptions](#subscriptions) | function                                                                  | no
-| [dispatch](#dispatch)           | [Dispatch](../architecture/dispatch.md)                                   | no
+| Prop                            | Type                                                                      | Required? |
+| ------------------------------- | ------------------------------------------------------------------------- | --------- |
+| [init](#init)                   | [State](../architecture/state.md) or [Action](../architecture/actions.md) | yes       |
+| [view](#view)                   | [View](../architecture/views.md)                                          | yes       |
```

Comment:
I'd like to keep the 💯.

Reply:
Not really! It's not accessible to just have an emoji. Can we have at least "Yes 💯"?